sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
+  COALESCE(protection_report, FALSE) AS protection_report,
+  COALESCE(pinned_tab, FALSE) AS pinned_tab,
+  COALESCE(default_browser, FALSE) AS default_browser,
+  COALESCE(many_uri, FALSE) AS many_uri,
```

Comment:
I would like to see each of these fields be named with a past-tense verb indicating an action that the user took, and for it to be as specific as possible. This becomes even more important as we turn this into a clients_last_seen-type table where we have to translate each of these names consistently into a `_bits` field name.

So this field would be `visited_10_uri` and the clients_last_seen field would be `visited_10_uri_bits`.

Other suggested names:

viewed_protection_report
used_pip
remembered_password

Outliers are these fields that don't indicate user actions, but rather represent configuration state, so these are more logical dimensions rather than usage indicators: 

default_browser
sync_configured

Are we considering these dimensions? Do we want to maintain 28 days of history for these, are would we just be keeping the most recently observed value? Can we make the distinction explicit?

Reply:
I updated these.  You make a good point that there is a difference between features that are used discretely and those that persist until "turned off".  Although `has_pinned_tab` is sort of in the middle as it persists until the tab is closed.

Anyway, I'd like to treat them all the same.  I still want to look for patterns of co-occurrence of feature use across types and be able to determine if the feature was used/enabled at all during some period.